icu4c/source/data/lang/en.txt

```diff
@@ -954,6 +954,13 @@ en{
             no{"Sort Digits Individually"}
             yes{"Sort Digits Numerically"}
         }
+        colReorder{
+            currency{"Currency"}
+            digit{"Digits"}
+            punct{"Punctuation"}
+            space{"Whitespace"}
+            symbol{"Symbol"}
+        }
```

Comment:
I assume this was the revert you mentioned

Reply:
No, this was for a separate issue, https://unicode-org.atlassian.net/browse/CLDR-15009. The revert I mentioned does not appear in this PR because it was reverted before I ever committed anything. It was a temporary change in CollationRuleParser::parseSetting to look up keywords using both "co" and "collation"; I needed to do that to get ICU to build at all, in order to be able to figure out what was really wrong. Once I figured out and fixed the real problem (in CLDR bcp47/timezone.xml), then I reverted the CollationRuleParser::parseSetting change (which never made it to a commit).